tests/Application.Tests.Integration/Borrows/Commands/ReturnDocumentTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Application.Borrows.Commands;
+using Application.Common.Exceptions;
+using Application.Identity;
+using Domain.Entities.Physical;
+using Domain.Statuses;
+using FluentAssertions;
+using Xunit;
+
+namespace Application.Tests.Integration.Borrows.Commands;
+
+public class ReturnDocumentTests : BaseClassFixture
+{
+    public ReturnDocumentTests(CustomApiFactory apiFactory) : base(apiFactory)
+    {
+    }
+
+    [Fact]
+    public async Task ShouldReturnBorrowRequest_WhenRequestIsValidAndBorrowRequestStatusIsCheckedOut()
+    {
+        // Arrange
+        var user = CreateUser(IdentityData.Roles.Employee, "abcdef");
+
+        var document = CreateNDocuments(1).First();
+        document.Status = DocumentStatus.Borrowed;
+        var borrow = CreateBorrowRequest(user, document, BorrowRequestStatus.CheckedOut);
+        await AddAsync(borrow);
+        var command = new ReturnDocument.Command()
+        {
+            DocumentId = document.Id
+        };
+        
+        // Act
+        var result = await SendAsync(command);
+        
+        // Assert
+        result.Id.Should().Be(borrow.Id);
+        result.Status.Should().Be(BorrowRequestStatus.Returned.ToString());
+        var documentResult = await FindAsync<Document>(result.DocumentId);
+        documentResult!.Status.Should().Be(DocumentStatus.Available);
+        result.BorrowTime.Should().NotBeAfter(borrow.BorrowTime.ToDateTimeUnspecified());
```

Comment:
I this necessary?

Reply:
Is*

Reply:
cường added this, not me

Reply:
@kaitoz11

Reply:
Wtf! Just to be sure man! Who knows when shit gonna break?
Imagine a sunny-day, some sleepy guy refactor this mo'ah fakka and casually delete a line of code.
If ya don't want to do it this way then we need to pull out another test method for that specific check just to test whether someone shit on it or not....

Reply:
fixed at https://github.com/ProFile-org/ProFile-BE/pull/206/commits/b5ec2c8e2d5b1941a0b1777b2fa202f95acbef5d